19_shamshaw/_targets.R

```diff
@@ -0,0 +1,91 @@
+library(targets)
+library(tarchetypes)
+
+source("src/prep_functions.R")
+source("src/plot_functions.R")
+
+options(tidyverse.quiet = TRUE,
+        clustermq.scheduler = "multiprocess")
+tar_option_set(packages = c("tidyverse", "lubridate", "scico", "paletteer","BAMMtools","scales", "ggforce"))
+
+list(
+  # Streamflow drought events calculated using Julian day 30-day moving window method 
+  # for 1980-2020 for gages in Colorado River Basin region. 
+  # Data file from regional drought early warning project 
+  # Data > Data from National Project > Streamflow > Drought Summaries
+  tar_target(events_crb_jd_1980_2020,
+             read_csv("data/weibull_jd_30d_wndw_Drought_Properties.csv")), 
```

Comment:
Is there any difference between this data and the 2021 data outside of the date range? Would it make sense to combine the two input data into a single target and then use that in the subsequent steps?

Reply:
and how comfortable are you with sharing this data? are we able to link to where the processing steps are done, or push it to s3 so someone could recreate your chart?

Reply:
I have the main figure pulling from the file that doesn't include 2021 because Caelan has done QA/QC on that file. The 2021 he hasn't done all the QA/QC so we just wanted to use that version to show events, but not do any specific data callouts. They might be 99% the same, but he just hasn't checked yet.

I can double check with John Hammond, but I think the preference is to not post those data files. I'll also double check on the processing steps - I believe those are just internal on gitlab right now

Reply:
OK, at a minimum we should provide a description in the `README` of the processing steps that happened from NWIS to here (e.g. how is a drought defined) and a site list for all of the gages used.

Reply:
Can do, I'll do up a readme for the data processing from NWIS